sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsublite/PubsubLiteWriteSchemaTransformProvider.java

```diff
@@ -141,12 +144,22 @@ public PCollectionRowTuple expand(PCollectionRowTuple input) {
   @AutoValue
   @DefaultSchema(AutoValueSchema.class)
   public abstract static class PubsubLiteWriteSchemaTransformConfiguration {
+    @SchemaFieldDescription(
+        "The GCP project where the Pubsub Lite reservation resides. This can be a "
+            + "project number of a project ID.")
     public abstract String getProject();
 
+    @SchemaFieldDescription("The region or zone where the Pubsub Lite reservation resides.")
     public abstract String getLocation();
 
+    @SchemaFieldDescription(
+        "The name of the topic to publish data into. This will be concatenated with "
+            + "the project and location parameters to build a full topic path.")
     public abstract String getTopicName();
 
+    @SchemaFieldDescription(
+        "The encoding format for the data stored in Pubsub Lite. Valid options are: "
+            + SUPPORTED_FORMATS_STR)
```

Comment:
Same comment here, I think the comma in the format string will need a space after it.